lib/flight_job/outputs/list_templates.rb

```diff
@@ -28,28 +28,27 @@
 require 'output_mode'
 
 module FlightJob
-  module Outputs::ListTemplates
-    extend OutputMode::TLDR::Index
-
-    register_column(header: 'Index', row_color: :yellow) do |template|
-      # NOTE: The OutputMode library does not support *_with_index type notation
-      #       Instead the index needs to be cached on the object itself
-      template.index
-    end
-    register_column(header: 'Name') do |template|
-      template.id
-    end
-    file_header = "File (Dir: #{FlightJob.config.templates_dir})"
-    register_column(header: file_header, verbose: true) do |template, interactive:|
-      if interactive
-        Pathname.new(template.workload_path).relative_path_from FlightJob.config.templates_dir
-      else
-        template.workload_path
+  class Outputs::ListTemplates < OutputMode::Formatters::Index
+    constructor do
+      register(header: 'Index', row_color: :yellow) do |template|
+        # NOTE: The OutputMode library does not support *_with_index type notation
+        #       Instead the index needs to be cached on the object itself
```

Comment:
What does this comment mean?  Why is it relevant here?  Everywhere else seems to have blocks such as `{ |object| object.foo }`.  Why does that need commenting on in this case?

Reply:
Yes that is correct, removed the comment

154ccf504649a3702ef33fcd7f85bf867ed7e0ec